packages/sdk/src/wallet/Ordit.ts

```diff
@@ -145,19 +145,16 @@ export class Ordit {
     }
   }
 
-  addAddress(type: AddressFormats, count = 1) {
+  getAddress(type: AddressFormats, accountIndex: number, addressIndex: number) {
```

Comment:
rename as per [BIP44 specification](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels)

```suggestion
  getAddress(type: AddressFormats, account: number, addressIndex: number) {
```